doc/lua_api.txt

```diff
@@ -1404,7 +1404,23 @@ Same as `image`, but does not accept a `position`; the position is instead deter
 * `world_pos`: World position of the waypoint.
 * `offset`: offset in pixels from position.
 
+### `compass`
 
+Displays an image oriented or translated according to current heading direction.
+
+* `size`: The size of this element.  Negative values represent that percentage
+  of the screen it should take; e.g. `x=-100` means 100% (width).
+* `scale`: Scale of the translated image (useless if rotation selected).
```

Comment:
When is this parameter needed, when is it superfluous? The phrasing is unclear to me.

Reply:
Tried to make it clearer :)